runtime/System/Console/Hawk/Runtime.hs

```diff
@@ -86,6 +90,8 @@ outputRows (OutputSpec _ spec) x = ignoringBrokenPipe $ do
     join "\n" = B.unlines
     join sep  = B.intercalate sep
 
+    copyFile i o = B.readFile i >>= \f -> B.writeFile o f
```

Comment:
`f` is a strange choice of variable name, since it is the _contents_ of the file, not a filename. I'd use `bs`, for `ByteString`.